cmd/osv-scanner/testmain_test.go

```diff
@@ -18,6 +20,210 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		panic(err)
 	}
+
+	// localmatcher.ZippedDBRemoteHost = testdb.NewZipDBCacheServer().URL
+	localmatcher.ZippedDBRemoteHost = testdb.NewZipDBCherryPickServer(map[string][]string{
```

Comment:
for now I just copied all the OSVs that were present in the snapshots for each database